$OpenBSD: patch-security_sandbox_common_SandboxSettings_h,v 1.3 2021/07/13 15:25:30 landry Exp $

https://phabricator.services.mozilla.com/D116633

include nsXULAppAPI.h for GeckoProcessType enum
In file included from /home/landry/src/m-c/security/sandbox/common/SandboxSettings.cpp:7:
/usr/obj/m-c/dist/include/mozilla/SandboxSettings.h:39:26: error: unknown type name 'GeckoProcessType'
bool StartOpenBSDSandbox(GeckoProcessType type);
Index: security/sandbox/common/SandboxSettings.h
--- security/sandbox/common/SandboxSettings.h.orig
+++ security/sandbox/common/SandboxSettings.h
@@ -7,6 +7,9 @@
 #define mozilla_SandboxSettings_h
 #include <cinttypes>
 
+#ifdef __OpenBSD__
+#include "nsXULAppAPI.h"
+#endif
 namespace mozilla {
 
 // Return the current sandbox level. This is the
